app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,7 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :configuration_script_authentications, :through => :configuration_scripts, :source => :authentications
```

Comment:
@mkanoor thanks for the input, I'll update this.

Right now the credentials are shared across `configuration_scripts` aka `job_templates`. This is what `configuration_script_authentications` links to. I might get away without this if I link all authentications to `ems` *and* share them